tools/README.md

```diff
@@ -0,0 +1,21 @@
+# Discovery Tools
+
+## Eaxmple:
+
+```
+$ go run tools/cmd/main.go --debug discovery example.com/app#1.0
```

Comment:
I don't see the point in setting `--debug` if it doesn't result in any debug logs.  If you want to clarify the distinction between debug logs and standard output, you can follow [this][1] [example][2] and store stderr in a temporary file.

[1]: https://github.com/xiekeyang/oci-discovery/blame/44ec3cf3113e29a743ad04220ccb7ff5197dab2a/README.md#L42
[2]: https://github.com/xiekeyang/oci-discovery/blame/44ec3cf3113e29a743ad04220ccb7ff5197dab2a/README.md#L67-L91